frontend/src/components/timerbar/timerbar.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
```

Comment:
We're using camelCase naming convention when it comes to naming folders and files. It would be great if you could change the file & folder name to timerBar instead of timerbar. (for both jsx and css files)